modules/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -83,3 +83,8 @@ public native function <ByteChannel channel> writeBytes(blob content,int startOf
 @Description { value:"Function to close a byte channel"}
 @Param {value:"channel: The ByteChannel to be closed"}
 public native function <ByteChannel channel> close();
+
+@Description {value:"Function to check whether next record available"}
+@Param {value:"channel: The TextRecordChannel to read text records from"}
+@Return {value:"Whether another record available or not"}
```

Comment:
Shall we rephrase the text in the `@Return` to the format followed in `contains()` in `string`, `hasNext()` in `datatable` etc.?